protocols/cosi/simulation.go

```diff
@@ -19,14 +19,19 @@ func init() {
 
 type CoSiSimulation struct {
 	sda.SimulationBFTree
+
+	// See https://github.com/dedis/cothority/issues/260
```

Comment:
I think the code should be self-contained (also documentation-wise). But as this is only in the simulation code, I don't really care that much.